.github/workflows/ci.yml

```diff
@@ -37,6 +37,24 @@ jobs:
 
       - name: Build debug APK and run jvm tests
         run: ./gradlew assembleDebug lintDebug testDebugUnitTest --stacktrace
+        
+      - name: Check for uncommitted changes
+        run: |
+          if [[ "$(git status --porcelain)" != "" ]]; then
+            echo ----------------------------------------
+            echo git status
+            echo ----------------------------------------
+            git status
+            echo ----------------------------------------
+            echo git diff
+            echo ----------------------------------------
+            git diff
+            echo ----------------------------------------
+            echo Troubleshooting
+            echo ----------------------------------------
+            echo "::error::Unstaged changes detected. Try locally cleaning your git repo (git clean -ffdx) and rebuild the app"
```

Comment:
Please add a note that this could be related to the kotlin auto-formatting.

Reply:
Updated the message